Pokemon Player/headers/Moving.h

```diff
@@ -0,0 +1,24 @@
+#ifndef MOVING_H
+#define MOVING_H
+
+#include <iostream>
+#include <fstream>
+#include "../headers/Control.h"
+
+class Moving {
+
+    public:
```

Comment:
Visibility keywords should be indented one space, and there shouldn't be a blank line between "class Moving" and "public:":
http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Class_Format